transmogrifier/sources/ead.py

```diff
@@ -331,15 +400,24 @@ def get_main_titles(cls, xml: Tag) -> list[str]:
         Args:
             xml: A BeautifulSoup Tag representing a single EAD XML record.
         """
-        return [
-            titleproper_value
-            for titleproper_elem in xml.find_all("titleproper")
-            if (
-                titleproper_value := ", ".join(
-                    string for string in titleproper_elem.stripped_strings
-                )
-            )
-        ]
+        collection_description = xml.metadata.find("archdesc", level="collection")
```

Comment:
I know this is repeated from `get_optional_fields` but I'm not sure if it's possible to pass this without significant changes to `Transformer`

Reply:
Yeah this seems fine.